packages/formula-editor/src/suggestion-menu.ts

```
@@ -0,0 +1,108 @@
+import { css, html, LitElement } from "lit";
+import { customElement, property } from "lit/decorators.js";
+
+@customElement("suggestion-menu")
+export class SuggestionMenu extends LitElement {
+
+  @property()
+  recommendations: string[] = [];
+
+  @property()
+  onClickRecommendation: Function = (recommendation: string) => {};
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)